src/site/stages/build/process-cms-exports/schemas/raw/node-page.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable camelcase */
```

Comment:
This file was accidentally brought over from @cvalarida's PR lol I did update line 9 to use `nodeStandardKeys`, however.